Dockerfile

```diff
@@ -1,27 +0,0 @@
-# Read about multistage images
```

Comment:
Please, don't remove it. We need it for deployment

Reply:
I created dev.Dockerfile instead of Dockerfile, or leave both files?